packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -422,6 +423,16 @@ function registerOrgPickerCommands(orgList: OrgList): vscode.Disposable {
   return vscode.Disposable.from(forceSetDefaultOrgCmd);
 }
 
+async function checkSObjectsRefreshed() {
+  const sobjectsCache = '.sfdx/typings/lwc/sobjects/**.d.ts';
```

Comment:
Is this the correct location? Is there a better way to find this than defining here?

Reply:
You can see how it's referenced in the location i mentioned [here](https://github.com/forcedotcom/salesforcedx-vscode/pull/2356#issuecomment-659751067)